conf/application.conf

```diff
@@ -98,6 +98,10 @@ scheduler {
     }
 }
 
+featureFlags {
+    pushPullNotificationFlag = true
```

Comment:
this may be called pushPullNotificationEnabled (we already know is a flag from the object name)

Reply:
done